packages/interface/package.json

```diff
@@ -0,0 +1,37 @@
+{
+	"name": "@wordpress/interface",
+	"version": "0.0.1",
+	"private": true,
```

Comment:
As long as public packages depend on it, it can't be private.

Reply:
Plugin packages are bundling it so I guess it can be private no?

Reply:
Would this mean third parties couldn't use it? Really eager for this :)

Reply:
Yes, for now, third parties would not be able to use this package. But it should end up being public when it is expanded and we are confident with its API :)

Reply:
I don't see why this package should be private if it's bundled. third-parties should be able to consume the package from npm and bundle it similarily to Gutenberg.

Reply:
And breaking changes in npm packages are allowed. They are forbidden for WordPress global APIs

Reply:
npm doesn't support bundling, it's only bundled from the webpack perspective or more precisely it isn't listed as an entry point. `@wordpress/edit-post` is a public package and `@wordpress/interface` is listed as a dependency so it needs to be exposed so people could install it.